packages/storyshots/stories/__test__/storyshots.test.js

```diff
@@ -1,2 +0,0 @@
-import initStoryshots from '../../src';
```

Comment:
The way they were tested was just a smoke test, which breaks now because the main index isn't executed within the context of that package's `package.json`